.github/workflows/publish.yml

```diff
@@ -37,13 +39,12 @@ jobs:
         run: npm ci
       - name: Build Website
         run: npm run build
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
The build process uses the `GITHUB_TOKEN` environment variable because needs to call the API several times. Would you mind to use the bot token here?